torch_ort_inference/pypi-readme.md

```diff
@@ -1,34 +1,16 @@
-The torch-ort packages uses the PyTorch APIs to accelerate PyTorch models using ONNX Runtime.
+The torch-ort-inference packages uses the PyTorch APIs to accelerate PyTorch models using ONNX Runtime OpenVINO EP.
```

Comment:
Should this be just a single "package" instead of "package(s)"

Reply:
Noted. Since we have only 1 package for torch-ort-inference unlike torch-ort.